Particle

wufan0311 opened the pull request.

super626 committed:
add ParticleUniverse

super626 committed:
abstract particle system

Commit:
add Align affector

Commit:
add part Affector

Commit:
Tab Format

super626 committed:
Merge pull request #159 from songchengjiang/particle

add Align affector

super626 committed:
modify folder, add files to xcode

Commit:
Adjust the directory structure of PU

super626 committed:
Merge pull request #164 from songchengjiang/particle

Adjust the directory structure of PU

Commit:
To complete the add Emitters

super626 committed:
Merge pull request #166 from songchengjiang/particle

To complete the add Emitters

super626 committed:
ios project and fix some warning

super626 committed:
add particle render

super626 committed:
particle render

super626 committed:
finish particle quad render

Commit:
merge to super626

super626 committed:
Merge pull request #167 from songchengjiang/particle

Link logic Level between System And affector(emitter)

Commit:
add particle test
render base particle

super626 committed:
extract abstract 3d particle system

super626 committed:
remove PU include and src folder

Commit:
add create func

Commit:
Merge branch 'particle' of https://github.com/super626/cocos2d-x into particle

Conflicts:
	extensions/3dparticle/CCParticle3DAffector.h
	extensions/3dparticle/CCParticle3DEmitter.cpp
	extensions/3dparticle/CCParticle3DEmitter.h
	extensions/3dparticle/CCParticleSystem3D.cpp
	extensions/3dparticle/CCParticleSystem3D.h
	extensions/3dparticle/ParticleUniverse/ParticleEmitters/CCParticle3DBoxEmitter.h
	extensions/3dparticle/ParticleUniverse/ParticleEmitters/CCParticle3DCircleEmitter.h
	extensions/3dparticle/ParticleUniverse/ParticleEmitters/CCParticle3DLineEmitter.h
	extensions/3dparticle/ParticleUniverse/ParticleEmitters/CCParticle3DMeshSurfaceEmitter.h
	extensions/3dparticle/ParticleUniverse/ParticleEmitters/CCParticle3DPointEmitter.h
	extensions/3dparticle/ParticleUniverse/ParticleEmitters/CCParticle3DPositionEmitter.h
	extensions/3dparticle/ParticleUniverse/ParticleEmitters/CCParticle3DSlaveEmitter.h
	extensions/3dparticle/ParticleUniverse/ParticleEmitters/CCParticle3DSphereSurfaceEmitter.h

Commit:
rename files

Commit:
add create func

Commit:
add depth sort
test on Android

Commit:
add ParticlePool

Commit:
test all affectors

zilongshanren commented:
@wufan0311 @super626 
This PR contains merge conflicts and it has some linker errors on Windows, please check.
thanks.

Commit:
add global particle

Commit:
finish first test demo

super626 committed:
Merge pull request #173 from songchengjiang/particle

finish first test demo

super626 committed:
ios work

super626 committed:
mac work

Commit:
add script compiler and translators

darkdukey commented:
Can you fix the conflicts?

Commit:
parse script

Commit:
modify model render

Commit:
fix model render BUG

super626 committed:
Merge pull request #176 from songchengjiang/particle

add script compiler and translators

Commit:
remove CCPUParticleSystem3DBuilder

super626 committed:
Merge pull request #177 from songchengjiang/particle

remove CCPUParticleSystem3DBuilder

super626 committed:
ios & mac works

super626 committed:
ios works

Commit:
add PUParticle3DQuadRender
fix bugs

Commit:
fix MeshCommand DepthWrite BUG

Commit:
Merge branch 'particle' of https://github.com/super626/cocos2d-x into particle

super626 committed:
Merge pull request #179 from songchengjiang/particle

Particle optimization

super626 committed:
config xcode project file

Commit:
fix setPosition setScale

super626 committed:
Merge pull request #183 from songchengjiang/particle

fix setPosition setScale invalid issues

Commit:
fix rotate issues
add demos

Commit:
set default scale to ONE

super626 committed:
Merge pull request #186 from songchengjiang/particle

fix rotate issues

super626 committed:
merge cocos

super626 committed:
Merge branch 'particle' of https://github.com/super626/cocos2d-x into particle

Commit:
fix uv animation BUG
add UVAnimation Demo

super626 committed:
add particle resource

Commit:
remove PUParticle3DModelRender

Commit:
add PUModelRender
add demos

Commit:
fix ScriptCompiler's memory leak
add new pu files
add create(pufile) interface in PUParticleSystem3D
add material ceche

Commit:
fix Particle3DRender memory leak

Commit:
fix memory lead

super626 committed:
Merge pull request #193 from songchengjiang/particle

Preload Material Files

super626 committed:
mac & ios work
